dali/operators/decoder/nvjpeg/nvjpeg_helper.h

```diff
@@ -71,6 +71,12 @@ class NvjpegError : public std::runtime_error {
         return "Error during the execution of the device tasks.";
       case NVJPEG_STATUS_IMPLEMENTATION_NOT_SUPPORTED:
         return "Not supported.";
+#if NVJPEG_VER_MAJOR > 11 || \
+    (NVJPEG_VER_MAJOR == 11 && (NVJPEG_VER_MINOR > 6 || \
+                               (NVJPEG_VER_MINOR == 6 && NVJPEG_VER_PATCH >= 0)))
```

Comment:
minor > 6 || (minor == 6 && patch >= 0)
Isn't that just equivalent to:
minor >=6
?

Reply:
True. Fixed.

Reply:
it seems to me it isn't. I read it all nvjpeg of "11.6.x" all minor 11.y.0  ie. releases with patch level 0 after minor 6"

Reply:
It was based on the code where the patch version was =! 0, but in this particular case, it is. So Rafal's suggestion should do.